transcom/mymove
# MAIN-B-21509

## [B-21509](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1018861)

[Int b 21509 PR #1](https://github.com/transcom/mymove/pull/14057#event-15353050251)

[Int b 21509 PR #2](https://github.com/transcom/mymove/pull/14255)

[Int b 21509 PR #3](https://github.com/transcom/mymove/pull/14323#event-15513197174)

## Summary

We are adding the `ports` and `port_location` tables to the database along with adding the Port of Debarkation (POD) and Port of Embarkation (POE) columns to `mto_service_items`. 

Also added tests for `ports` and `port_location` and updated `mto_service_items` tests.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Run the migrations:  
   - I ran mine using `make db_dev_reset db_dev_migrate`, though the reset might not be necessary.
   - You could also run `make server_test`
2. Verify that the tables and new columns have been added.
   - I verified mine using DBeaver.


### Database

#### Any new migrations/schema changes:
